psse/psse-model/src/main/java/com/powsybl/psse/model/PsseRawReader.java

```diff
@@ -21,39 +23,255 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
  */
 public class PsseRawReader {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PsseRawReader.class);
 
-    private static String removeComment(String line) {
-        int slashIndex = line.lastIndexOf('/');
-        if (slashIndex == -1) {
-            return line;
+    public boolean checkCaseIdentification(BufferedReader reader) throws IOException {
```

Comment:
Not sure that this should be public. If you make this method private, you don't have to check that reader is not null: this will be done by another public method

Reply:
Now should be public. Next PR will refact and extend the validation process and at this moment I will try to change the visibility to private